minimap.impl.user.js

```diff
@@ -574,8 +683,7 @@ const { html, render } = mlp_uhtml;
         .shadowRoot.querySelector("mona-lisa-color-picker")
         .shadowRoot.querySelector("button.confirm");
       if (placeButton && diff.length > 0) {
-        const randID = Math.floor(Math.random() * diff.length);
-        const randPixel = diff[randID];
+        const randPixel = selectRandomPixel(diff);
```

Comment:
If you could refactor `selectRandomPixel(diff)` to return an object with `.x` and `.y` properties and then use those below, that'd be lovely.

Alternatively, have it destructure out some `x` and `y` bindings like you do above.